spec/models/occupation_standard_spec.rb

```diff
@@ -83,4 +83,20 @@
       expect(occupation_standard.competencies_count).to eq 3
     end
   end
+
+  describe ".by_title" do
+    it "returns records that match the argument in title" do
+      first_occupation = create(:occupation_standard, title: "AAAAAA")
+      create(:occupation_standard, title: "ZZZZZZ")
+
+      expect(described_class.by_title("A").pluck(:id)).to match_array([first_occupation.id])
+    end
+
+    it "returns all records if title not provided" do
+      first_occupation = create(:occupation_standard, title: "AAAAAA")
+      second_occupation = create(:occupation_standard, title: "ZZZZZZ")
+
+      expect(described_class.by_title("").pluck(:id)).to match_array([first_occupation.id, second_occupation.id])
+    end
```

Comment:
Could we add another spec or two that handles two word titles, like "pipe fitter"?